knowledge-base/grid-handle-empty-popup-footer.md

```diff
@@ -0,0 +1,460 @@
+---
+title:  How to handle the empty popup footer when using the Popup Form Template
```

Comment:
* "Handling" = "How to handle" but shorter
* Use title case
```suggestion
title:  Handling the Empty Popup Footer when Using the Popup Form Template
```

Reply:
In addition, "handle" seems too generic. Why not use a keyword that a customer would search with, e.g. "hide" or "remove"

Reply:
Updated in https://github.com/telerik/blazor-docs/pull/2414/commits/783d5e16f8d98b77f5d5d01c81fe7708f1fc69ba